src/main/java/core/basesyntax/service/FigurePainter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FigurePainter {
+    public void draw();
```

Comment:
all interface methods are public by default
```suggestion
    void draw();
```